Feature/data model

cliffxuan opened the pull request:
1. initial data model
2. minimal api client for float
   ![entity-diagram](https://cloud.githubusercontent.com/assets/640204/15215085/7aeaa73c-1848-11e6-95cd-514b2fcd82c2.png)

cliffxuan committed:
minimal api client for float

cliffxuan committed:
initial models

cliffxuan committed:
added timeline for project

s-block commented on dashboard/apps/prototype/models.py:
```diff
@@ -1,3 +1,87 @@
+    amount = models.DecimalField(max_digits=5, decimal_places=2)
+    person = models.ForeignKey('Person', related_name='rates')
+    start_date = models.DateTimeField()
+    end_date = models.DateTimeField()
```
maybe set null=True on end_date as an end_date won't be know

you could even get rid of end_date all together as the end date will be the start date of the later entry

cliffxuan replied:
good point. going to remove the `end_date`

s-block commented on dashboard/apps/prototype/models.py:
```diff
@@ -1,3 +1,87 @@
+    person = models.ForeignKey('Person')
+    role = models.ForeignKey('Role')
+    start_date = models.DateTimeField()
+    end_date = models.DateTimeField()
```
end_date for the role won't be known when the PersonRole is created so would have to be null=True

Or remove end_date and use the start_date of the later record as the end_date

cliffxuan replied:
good point. will remove `end_date`

cliffxuan committed:
more fields fiddling

cliffxuan commented:
updated diagram. added a few `float_id`s and removed some `end_date`s.

![entity-diagram](https://cloud.githubusercontent.com/assets/640204/15218439/daa8c320-1857-11e6-9427-0fc7a9705780.png)

cliffxuan committed:
allow params

cliffxuan committed:
more column modifications

cliffxuan commented:
latest
![entity-diagram](https://cloud.githubusercontent.com/assets/640204/15284314/767788f2-1b48-11e6-922d-c6d1baee3638.png)
